### PR TITLE
VDS-2904, VDS-2908: Import Physical products

### PR DESCRIPTION
**Pull request by asvishnyakov**
Description: 

--

QA-test: VDS-2906

Demo-test: VDS-2907

Download artifact URL: https://vc3prerelease.blob.core.windows.net/packages/VirtoCommerce.CatalogExportImport_1.1.0-pr-18-779099e9.zip

### CODE REVIEW

File: src/VirtoCommerce.CatalogExportImportModule.Data/Services/CsvPagedPhysicalProductDataImporter.cs [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+            IImportProductSearchService importProductSearchService, IImportCategorySearchService importCategorySearchService,
+            IItemService itemService)
+            : base(dataSourceFactory, importRecordsValidator, importReporterFactory, blobUrlResolver)
+        {
+            _importProductSearchService = importProductSearchService;
+            _importCategorySearchService = importCategorySearchService;
+            _itemService = itemService;
+        }
+
+        public override string DataType { get; } = nameof(CatalogProduct);
```
Comment (trueboroda) [inline src/VirtoCommerce.CatalogExportImportModule.Data/Services/CsvPagedPhysicalProductDataImporter.cs]: Use the module's constant DataTypes.PhysicalProduct instead nameof()

File: src/VirtoCommerce.CatalogExportImportModule.Data/Services/CsvPagedPhysicalProductDataImporter.cs [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+                HandleError(progressCallback, importProgress, e.Message);
+            }
+            finally
+            {
+                importProgress.ProcessedCount = Math.Min(dataSource.CurrentPageNumber * dataSource.PageSize, importProgress.TotalCount);
+                importProgress.ErrorCount = importProgress.ProcessedCount - importProgress.CreatedCount - importProgress.UpdatedCount;
+            }
+        }
+
+        private async Task<CatalogProduct[]> SearchEntitiesByIdAndOuterIdAsync(string[] internalIds, string[] outerIds)
```
Comment (trueboroda) [inline src/VirtoCommerce.CatalogExportImportModule.Data/Services/CsvPagedPhysicalProductDataImporter.cs]: Maybe rename to smth with 'Products'

**Review by trueboroda — CHANGES_REQUESTED** [review]

File: src/VirtoCommerce.CatalogExportImportModule.Data/Services/CsvPagedPhysicalProductDataImporter.cs (hunk 2 above)
Comment (asvishnyakov) [reply src/VirtoCommerce.CatalogExportImportModule.Data/Services/CsvPagedPhysicalProductDataImporter.cs]: Fixed

File: src/VirtoCommerce.CatalogExportImportModule.Data/Services/CsvPagedPhysicalProductDataImporter.cs (hunk 1 above)
Comment (asvishnyakov) [reply src/VirtoCommerce.CatalogExportImportModule.Data/Services/CsvPagedPhysicalProductDataImporter.cs]: Fixed

**Comment by pushnitsa** [comment]
Please add the correct title like here:
image

**Review by pushnitsa — APPROVED** [review]

**Pull request merged**